includes/apple-push-api/request/class-request.php

```diff
@@ -265,6 +265,10 @@ private function parse_response( $response, $json = true, $type = 'post', $meta
 				$message = __( 'Unable to fetch information for the specified article. Was it deleted?', 'apple-news' );
 			}
 
+			if ( 'DUPLICATE_ARTICLE_FOUND' === $error->code ) {
+				$message .= '. Original UUID: ' . $error->value;
```

Comment:
Tried to add a newline (`\n`) in here for formatting clarity but didn't carry over to the frontend luck.  Open to other more readable templating/verbiage suggestions.

Reply:
Use a `<br>`, see if that works with the output escaping we're using

Reply:
No luck, `<br>` is getting escaped as well.  Also tried passing string to [nl2br()](https://www.php.net/manual/en/function.nl2br.php).

Reply:
Realized I was testing classic editor php output against the block editor/GB frontend notices.  Had to include [this commit](https://github.com/alleyinteractive/apple-news/pull/971/commits/80968f07797d0086e415664651cff48b3b1a5f10) to allow `<br>`s in the block editor.

Also modified the wp_kses allowed html like you suggested to get line breaks working in the classic editor too.